force-app/main/default/triggers/Count.trigger

```diff
@@ -0,0 +1,11 @@
+trigger Count on Task(after insert, after update, after delete) {
```

Comment:
There is just one Trigger per sObject and the trigger should be called as the sObject, i.e. TaskTrigger